packages/contact-center/store/src/store.types.ts

```diff
@@ -0,0 +1,31 @@
+import {AgentLogin, IContactCenter, Profile, Team} from '@webex/plugin-cc';
+
+interface WithWebex {
+    webex: { cc: IContactCenter };
+}
+
+interface WithWebexConfigAndToken {
```

Comment:
We can simplify the name of this interface: WebexConfig or StoreConfig should be ok

Reply:
This can be an internal type as the type that is exported is initParams. But im okay with making the name simpler also.

Reply:
It is internal and kept it so just to make the InitParams type to be descriptive:

```
type InitParams = WithWebex | WithWebexConfigAndToken;
```

Changed it to `WithWebexConfig` considering the token as part of the config anyway